docker/test/integration/cluster/containers/CouchbaseServerContainer.py

```diff
@@ -47,7 +47,7 @@ def deploy(self):
             return
 
         self.docker_container = self.client.containers.run(
-            "couchbase:community-7.6.2",
+            "couchbase:enterprise-7.2.5",
```

Comment:
Yes, it is stated on it's dockerhub page that it can be used freely for testing purposes:
"Enterprise Edition -- The Enterprise Edition license provides for free for development and testing for Couchbase Enterprise Edition."